controls/V-72901.rb

```diff
@@ -20,7 +20,7 @@
 -----------------
 =end
 PG_VERSION = attribute(
-  'pg_version',
+  'pg_versionsion',
```

Comment:
Typo ?

Reply:
Copy-paste error; this has been corrected.